config/standalone-config.properties

```diff
@@ -10,38 +10,64 @@
 
 ##region General Segment Store Settings
 
-# Port on which a ZK server is started
+# Port on which a ZK server is started.
+# Default value: 4000
```

Comment:
According to other `.properties` files (and their respective `XXXConfig.java`), the value set here is assumed to be the default value. I don't know if we need an additional note on each field specifying that this is the default.

Reply:
For some of the properties in config.properties, we have a note that explicitly specify the default value. For example: 
```
...
# Default value: BOOKKEEPER
pravegaservice.dataLogImplementation=BOOKKEEPER
...
# Default value: HDFS
# pravegaservice.storageImplementation=HDFS
```

The note might turn out to be useful sometimes if the user has modified the value and wants to go back to the default.

What do you say?